ios/Podfile

```diff
@@ -1,13 +1,14 @@
-require_relative '../node_modules/react-native/scripts/react_native_pods'
-require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+# Resolve react_native_pods.rb with node to allow for hoisting
+require Pod::Executable.execute_command('node', ['-p',
+  'require.resolve(
+    "react-native/scripts/react_native_pods.rb",
+    {paths: [process.argv[1]]},
+  )', __dir__]).strip
+platform :ios, min_ios_version_supported
```

Comment:
Where does `min_ios_version_supported` come from now?

Reply:
But we want to set our own, not be tied to whatever RN decides, even though we need to take it into consideration.

Reply:
Oh, I see.